tests/console/zdup_prepare_leap_sle.pm

```diff
@@ -0,0 +1,96 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: prepare migration from Leap to SLE
+#   Documentation for 15:
+#   https://www.suse.com/de-de/documentation/sles-15/book_sle_upgrade/data/sec_upgrade-online_opensuse_to_sle.html
+#   12 is not supported and inoffical
+# Maintainer: lemon.li <leli@suse.com>
+
+use base "installbasetest";
+use registration qw(scc_version add_suseconnect_product);
+use testapi;
+use utils 'zypper_call';
+use strict;
+use warnings;
+
+#   Function: judge base system is which version of leap
```

Comment:
I suggest to use a common POD format as in other files, e.g. take `lib/utils.pm`. Also I struggle to understand the sentence. How about "Check for a specific base version of openSUSE Leap"?

Reply:
Updated, thanks.